packages/typography/src/Typography.story.tsx

```diff
@@ -24,6 +33,23 @@ storiesOf('Typography', module).add('Default', () => (
       <InlineCode className={displayBlock}>InlineCode</InlineCode>
       <Disclaimer className={displayBlock}>Disclaimer</Disclaimer>
       <Overline>Overline</Overline>
+      <Link
```

Comment:
Can we add a knob to show or hide the external link to demonstrate that behavior?

Reply:
That's why the href is selectable

Reply:
Oh whoops. How would you disable the external link icon in rare cases where that's necessary? I believe it's something we identified as needing a prop for before.

Reply:
I didn't have a prop for this, as I thought this was part of the design guidelines. I'm happy to add a prop, maybe `externalIconAppearance`?

Reply:
Rob and I spoke about this one, and acknowledged there will be cases where it wouldn't be able to be shown. It's not noted in the guidelines, but the cases do exist.

Reply:
Added support for this, let me know what you think.

Reply:
Looks good!